docs/src/P3Scheme.md

```diff
@@ -429,10 +429,306 @@ When modifying process rates, we now need to consider whether they are concerned
+      + F_{rim}(\textcolor{lime}{ - \text{QISUB}} 
+                \textcolor{pink}{ - \text{QIMLT}})
+                \\
+                 &\quad
+                  \textcolor{brown}{ + \text{QCCOL} + \text{QRCOL} + \text{QIWET}}
+                \\
+  S_{L_{ice}} &=
+                  \textcolor{orange}{\text{QCHET} + \text{QRHET} + \text{QCHOM} + \text{QRHOM}}
+                  \textcolor{lime}{ - \text{QISUB}}
+                  \textcolor{pink}{ - \text{QIMLT}}
+                \\
+                 &\quad
+                \textcolor{brown}{   + \text{QCCOL} + \text{QRCOL} + \text{QIWET}}
+                \textcolor{lime}{    + \text{QIDEP}}
+                \textcolor{magenta}{ + \text{QINUC}}
+                \\
+  S_{N_{ice}} &= 
+               \textcolor{orange}{   \text{NCHET} + \text{NRHET} + \text{NCHOM} + \text{NRHOM}} 
+               \textcolor{lime}{   - \text{NISUB}} 
+               \textcolor{magenta}{+ \text{NINUC} }
+               \\
+  S_{B_{rim}} &= 
+                \frac{1}{ρ^*} (
+                  \textcolor{orange}{\text{QCHET} + \text{QRHET} + \text{QCHOM} + \text{QRHOM}}) 
+              + \frac{1}{ρ_{rim}} F_{rim}(
+                  \textcolor{lime}{ - \text{QISUB}} 
+                  \textcolor{pink}{ - \text{QIMLT}})
+               \\
+                &\quad 
+               \textcolor{brown}{+ \text{BCCOL} + \text{BRCOL} + \text{BIWET}} 
+                \\
+\end{align*}
+```
+
+Terms that are related are colored the same:
+
+- The $\textcolor{cyan}{\textsf{cyan}}$ colors → warm rain microphysics (i.e. WRM)
+  - nucleation, condensation, autoconversion, accretion, evaporation
+- The $\textcolor{orange}{\textsf{orange}}$ colors → heterogeneous and homogeneous freezing (i.e. HET, HOM)
+- The $\textcolor{brown}{\textsf{brown}}$ colors → liquid-ice collisions (i.e. COL, SHD, WET)
+- The $\textcolor{pink}{\textsf{pink}}$ colors → melting (i.e. MLT)
+- The $\textcolor{lime}{\textsf{lime}}$ colors → deposition/sublimation (i.e. DEP, SUB)
+- The $\textcolor{magenta}{\textsf{magenta}}$ colors → nucleation (i.e. NUC)
+
+!!! note "Differences with Morrison & Milbrandt (2015)"
+    There are numerous differences between the P3 scheme in Morrison & Milbrandt (2015) and the P3 scheme in this package:
+    - the maximum freezing rate (wet growth limit) is computed at the particle level, instead of as a bulk process
+    - changes in rime volume due to (dry) collisions with cloud and rain are computed at the particle level, with a local rime density evaluated as a function of both the liquid and ice particle diameters
+    - in the wet growth regime, the densification process is computed as a rapidly adjusting bulk process, instead of instantenous particle-level densification. We also scale by the fraction of the mass rate that undergoes wet growth.
+
+
+Below, we describe the different processes in more detail.
+
+### Collisions with liquid droplets
```

Comment:
Could we add in parenthesis which of the QRCOL, BIWET etc acronyms are covered in each sub-section?